include/boost/geometry/algorithms/detail/distance/point_to_geometry.hpp

```diff
@@ -102,7 +104,8 @@ class point_to_range
     static inline return_type apply(Point const& point, Range const& range,
                                     Strategy const& strategy)
     {
-        return_type const zero = return_type(0);
+
+        return_type const zero = return_type();
```

Comment:
Ok, so now user-defined `return_type` is required to have a default ctor initializing it to 0. Is that correct?

Reply:
Yes, I changed it by adding a constructor to the return_type of `closest_points`